filters/style_manager.py

```diff
@@ -114,7 +114,14 @@ def responseComplete(self):
             style_manager = qgis_layer.styleManager()
             if '' in style_manager.styles():
                 style_manager.renameStyle('', 'default')
-            project.write()
+                project.write()
+                # Make sure properties are saved.
+                # So GetCapabilities are working.
+                project.clear()
```

Comment:
I should be the one who says sorry :p. This patches came from me anyway.

I don't know why, somehow if I don't call project.clear(), GetCapabilities don't return any styles if I already set default style name to `default`